File: 240310/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

    char a;
    cin>>a;

    string str[5]={"apple","banana","grape","blueberry","orange"};

int cnt=0;

for(int i=0;i<5;i++){
    if(str[i][2]==a||str[i][3]==a){
        cout<<str[i]<<endl;
        cnt++;
    }
}
cout<<cnt;
return 0;
}

File: 240614/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    Scanner sc=new Scanner(System.in);

    char c=sc.next().charAt(0);
    int cnt=0;

    String [] str=new String[]{"apple","banana","grape","blueberry","orange"};

    for(int i=0;i<5;i++)
    {
        if(str[i].charAt(2)==c||str[i].charAt(3)==c){
            System.out.println(str[i]);
            cnt++;
        }
    }
    System.out.println(cnt);
    }
}